scripts/fetch_external_files.sh

```diff
@@ -3,8 +3,6 @@ root_dir=$1
 integration=$2
 integration_dir="${root_dir}/exporters/${integration}"
 
-repository_definitions="https://github.com/newrelic-experimental/entity-synthesis-definitions"
-
```

Comment:
could you check if `definition_files_dir` in line 12 is related to synthesis definitions?

Reply:
it's a legacy folder that was used before deciding to embed the definition in the nri-prometheus config, so I am deleting it.

Reply:
Done, I have checked adnd there were still more stuff to delete, now it's correct.